app/client/ui/UserManager.ts

```diff
@@ -103,13 +106,13 @@ export function showUserManagerModal(userApi: UserAPI, options: IUserManagerOpti
     if (model.isSelfRemoved.get()) {
       const name = resourceName(model.resourceType);
       confirmModal(
-        `You are about to remove your own access to this ${name}`,
-        'Remove my access', tryToSaveChanges,
+        t(`You are about to remove your own access to this {{name}}`, { name }),
+        t('Remove my access'), tryToSaveChanges,
         {
           explanation: (
-            'Once you have removed your own access, ' +
-            'you will not be able to get it back without assistance ' +
-            `from someone else with sufficient access to the ${name}.`
+            t(`Once you have removed your own access, \
+            you will not be able to get it back without assistance \
```

Comment:
Here too, it may be better to remove the leading whitespace on continuation lines, even if code ends up less well-aligned.